enterprise/cloud.oracle/external/binaries-list

```diff
@@ -0,0 +1,54 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+19F36E3A7A4914A5B361F840D65EF00B670CE064 com.oracle.oci.sdk:oci-java-sdk-common:2.10.0
+3D4C9B62BDDFC86B154C78FA751257D6B2BB309A com.oracle.oci.sdk:oci-java-sdk-database:2.10.0
+9DE4EA6F96CFFB2C7EEDFA903AAA7A5A2D345F18 com.oracle.oci.sdk:oci-java-sdk-identity:2.10.0
+992769EA9FB96F0CB65B65B63DFCF785DEFFFC5F com.oracle.oci.sdk:oci-java-sdk-circuitbreaker:2.10.0
+F00ECE576884E10D5F2E4B34D1D1785ADE41211C com.oracle.oci.sdk:oci-java-sdk-workrequests:2.10.0
+CACECE66244150F252B5C58E0A38FEA0D5015665 com.nimbusds:nimbus-jose-jwt:9.11.1
+25EA2E8B0C338A877313BD4672D3FE056EA78F0D com.google.code.findbugs:jsr305:3.0.2
+BA035118BC8BAC37D7EFF77700720999ACD9986D com.google.j2objc:j2objc-annotations:1.3
+6505A72A097D9270F7A9E7BF42C4238283247755 org.apache.commons:commons-lang3:3.8.1
+4BFC12ADFE4842BF07B657F0369C4CB522955686 commons-logging:commons-logging:1.2
+49D94806B6E3DC933DACBD8ACB0FDBAB8EBD1E5D commons-codec:commons-codec:1.15
```

Comment:
FYI - some of the libraries (commons-lang, commons-codec) also appears as wrapped libraries and/or in other modules. Maybe we could upgrade the shipped wrapped libraries or share the libraries between modules ?

jackson seems also part of other enterprise module, bouncycastle has its own wrapped already.

Reply:
+1 we want to share the libraries, if possible.